modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -107,16 +107,20 @@ export default {
     editorOptions() {
       const activeOptions = Object.assign({}, this.options);
 
-      // Allow toolbar option to pass through if `false`
-      activeOptions.toolbar = (activeOptions.toolbar !== undefined)
-        ? activeOptions.toolbar : this.defaultOptions.toolbar;
-
       activeOptions.styles = this.enhanceStyles(
         activeOptions.styles?.length
           ? activeOptions.styles
           : this.defaultOptions.styles
       );
 
+      // Allow default options to pass through if `false`
+      for (const option in this.defaultOptions) {
```

Comment:
Iterate over `Object.entries(this.defaultOptions)` to avoid any possible issues with inherited property names (the `for...in` statement is a little bit dangerous).

Reply:
Because I need the keys of `this.defaultOptions ` I am going with `Object.keys(this.defaultOptions).forEach(...)`. I hope this is fine.

Also, if you have the time I would like to know how the `for ... in` statement is dangerous.